src/Microsoft.Health.Fhir.Core/Features/Operations/BulkDelete/BulkDeleteDefinition.cs

```diff
@@ -56,5 +61,11 @@ protected BulkDeleteDefinition()
 
         [JsonProperty(JobRecordProperties.ParentRequestId)]
         public string ParentRequestId { get; private set; }
+
+        [JsonProperty(JobRecordProperties.StartingResourceCount)]
+        public long StartingResourceCount { get; private set; }
```

Comment:
Question not suggestion - would `ExpectedResourceCount` be better here? New to bulk delete and it seems like this is being used to hold how many resources we expect to see deleted. I `a)` could be wrong or `b)` continue my streak of bad name suggestions. 😂

Reply:
You are correct, and that name I think could also work. It probably does fit better.

Reply:
Changed